src/containers/Profile/Profile.jsx

```diff
@@ -0,0 +1,167 @@
+import React from "react";
+import { connect } from "react-redux";
+import { firestoreConnect } from "react-redux-firebase";
+import { compose } from "redux";
+import { Link, Redirect } from "react-router-dom";
+import { Tabs, Tab, Container } from "react-bootstrap";
+import anonymousImage from "../../images/anonymousImage.png";
+import profileHeaderBackground from "../../images/profileHeaderBackground.png";
+import Header from "../components/common/Header";
+import "./Profile.css";
+import AccountSettings from "./ProfileSections/AccountSettings/AccountSettings";
+import ManageUsers from "./ProfileSections/ManageUsers/ManageUsers";
+import ManageBlogs from "./ProfileSections/ManageBlogs/ManageBlogs";
+import { updateProfilePhoto } from "../../actions/authActions";
+
+const Profile = (props) => {
+  const { profile, auth, id, currentUserType } = props;
+  const ADMIN_ROLE = "ADMIN";
```

Comment:
move to line 15, these constants usually go outside of the function (this helps indicate to readers that it has no dependencies on things inside the scope fo the fn)